.claude/skills/twelve-factor-methodology/agents/agent.factor-check.sh

```
@@ -0,0 +1,153 @@
+    status="warn"
+    details="No docker-compose.yml for dev environment"
+  fi
+
+  echo "{\"factor\":\"X. Dev/Prod Parity\",\"status\":\"$status\",\"details\":\"$details\"}"
+}
+
+# ============================================================================
+# Main execution
+# ============================================================================
+RESULTS=()
+RESULTS+=($(check_factor_i_codebase))
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)